### PR TITLE
Add paragraph about Arcana game engine

### PR DESCRIPTION
**Pull request by zakarumych**
I hope it isn't too late to add note for my upcoming game engine.

### CODE REVIEW

File: content/news/021/index.md [hunk 1]
```diff
@@ -466,6 +466,44 @@ the recent engine updates:
 [rg3d_twitter]: https://twitter.com/DmitryNStepanov
 [rg3d_wasm_demo]: https://rg3d.rs/assets/webexample/index.html
 
+### [Arcana]
+
+Arcana is new game engine built with focus on ease of use
+without compromising on level of control.\
+The engine is aimed to support wide variety of games,
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:473]:
```suggestion
The engine is aimed to support a wide variety of games,
```

File: content/news/021/index.md [hunk 2]
```diff
@@ -466,6 +466,44 @@ the recent engine updates:
...
+from pixel-art to fully ray-traced,
+from single-player puzzles to online strategies.
+
+It is at very early stage, not all necessary subsystems are done
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:477]:
```suggestion
It is at a very early stage, not all necessary subsystems are done
```

File: content/news/021/index.md [hunk 3]
```diff
@@ -466,6 +466,44 @@ the recent engine updates:
...
+and code is in flux.
+
+The engine already can be used to build demos like this:
+
+![arcana](tanks.gif)
+
+This particular demo has been coded in single evening,\
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:484]:
```suggestion
This particular demo has been coded in a single evening,
```

File: content/news/021/index.md [hunk 4]
```diff
@@ -466,6 +466,44 @@ the recent engine updates:
...
+so all sprites are rendered in single instanced draw call,
+allowing rendering millions of sprites in one frame.
+
+Rendering is done with [`sierra`] - Vulkan-like graphics API with
+batteries included.
+
+Arcana uses [`hecs`] as ECS and rolls its own simplistic `System` trait
+to define and run systems, once per frame or with fixed steps.
+
+[`rapier`] physics is integrated for 2D case, but system is kept opt-in.
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:498]:
```suggestion
[`rapier`] physics is integrated for 2D cases, but system is kept opt-in.
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
The content looks good for the most part!

The only major issue is that the GIF is 15mb, which is a long way over our usual limit of 2.5mb. I tried to optimize the GIF via [EZGif](https://ezgif.com/optimize/e), but I couldn't get it any lower than 4mb on my end. Could you provide a smaller GIF, or replace it with a screenshot?

**Comment by zakarumych** [comment]
I'll record shorter video and try to fit 2.5mb

File: content/news/021/index.md [hunk 5]
```diff
@@ -466,6 +466,44 @@ the recent engine updates:
...
+It is at a very early stage, not all necessary subsystems are done
+and code is in flux.
+
+The engine already can be used to build demos like this:
+
+![arcana](tanks.gif)
+
+This particular demo has been coded in a single evening,
+together with sprite sheet loading and sprite animations
+which will be integrated into the engine later.
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:486]:
```suggestion
![arcana](tanks.gif)
_A demo Arcana game_

Arcana is a new game engine built with focus on ease of use
without compromising on level of control.
The engine is aimed to support a wide variety of games,
from pixel-art to fully ray-traced,
from single-player puzzles to online strategies.

It is at a very early stage, not all necessary subsystems are done
and code is in flux.

The demo shown above was coded in a single evening,
together with sprite sheet loading and sprite animations
which will be integrated into the engine later.
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: Looks good, thank you for fixing the GIF! I've made one tweak to the ordering of the post to fit with our usual format (screenshots first, text second), think this is ready to merge once I've committed that.

File: content/news/021/index.md (hunk 5 above)
Comment (zakarumych) [reply content/news/021/index.md]: Yeah, seems great!

File: content/news/021/index.md [hunk 6]
```diff
@@ -466,6 +466,43 @@ the recent engine updates:
 [rg3d_twitter]: https://twitter.com/DmitryNStepanov
 [rg3d_wasm_demo]: https://rg3d.rs/assets/webexample/index.html
 
+### [Arcana]
+
+![arcana](tanks.gif)
+_A demo Arcana game_
+
+Arcana is a new game engine built with focus on ease of use
+without compromising on level of control.
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:475]: Adding an extra link:
```suggestion
[Arcana] is a new game engine built with focus on ease of use
without compromising on level of control.
```

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR!

**Pull request merged**